src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -67,4 +67,14 @@ public boolean equals(Object other) {
                 || (other instanceof AddCommand // instanceof handles nulls
                 && toAdd.equals(((AddCommand) other).toAdd));
     }
+
+    @Override
+    public String toString() {
+        return "Add";
+    }
+
+    @Override
+    public String getCommandUsage() {
+        return MESSAGE_USAGE;
+    }
```

Comment:
These methods are not used at all, left over from when you were trying out various implementations. Our test coverage took a huge hit as a result, so I think we should remove these unused methods.